arcaflow_plugin_opensearch/opensearch_plugin.py

```diff
@@ -9,6 +9,46 @@
 from opensearch_schema import ErrorOutput, SuccessOutput, StoreDocumentRequest
 
 
+def convert_to_supported_type(value) -> typing.Dict:
```

Comment:
The type hints on this signature look wrong:  it should accept an `Any` and return an `Any`.